src/PHPQueue/Backend/Predis.php

```diff
@@ -6,7 +6,19 @@
 use PHPQueue\Interfaces\FifoQueueStore;
```

Comment:
Nice!! Just a few minor comments:
- Maybe be a little more consistent with terminology? For example, in the PHPdoc on line 11, you talk about a key-value store, but in the exception messages in push() you use "zset".
- A good approach might be to mention in the PHPdoc that this is backed by a Redis zset... and also maybe note that that's the origin of the term "score".
- A bit more inline comments explaining the why of some stuff would be cool, I think... Since I'm not looking at the whole codebase I'm not sure what's obvious and what's not, but... it does feel like there could be more inline explanations.